client/src/components/Property.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import Modal from 'plaid-threads/Modal';
+import ModalBody from 'plaid-threads/ModalBody';
+import Button from 'plaid-threads/Button';
+import TextInput from 'plaid-threads/TextInput';
+import PropTypes from 'prop-types';
+
+import { useProperties } from '../services';
+
+Property.propTypes = {
+  userId: PropTypes.number,
+};
+
+export default function Property({ userId }) {
+  const [show, setShow] = useState(false);
+  const [description, setDescription] = useState('');
+  const [value, setValue] = useState('');
+  const { addProperty } = useProperties();
+
+  const handleSubmit = () => {
+    setShow(false);
+    addProperty(
+      userId,
+      description,
+      parseFloat(value.replace(/[^0-9.-]+/g, ''))
```

Comment:
i think if you use a `NumberInput` instead of a `TextInput`, you wont have to do this. you can just `parseInt`